packages/rrweb-snapshot/src/utils.ts

```diff
@@ -164,41 +164,61 @@ export function createMirror(): Mirror {
   return new Mirror();
 }
 
-export function maskInputValue({
-  element,
+export function shouldMaskInput({
   maskInputOptions,
   tagName,
   type,
+}: {
+  maskInputOptions: MaskInputOptions;
+  tagName: Uppercase<string>;
+  type: Lowercase<string> | null;
+}): boolean {
+  // Handle `option` like `select
+  if (tagName === 'OPTION') {
+    tagName = 'SELECT';
+  }
+  const actualType = type && toLowerCase(type);
```

Comment:
`type`should already be forced to a lower case string so I guess no need to convert it to lowercase again here?